static/js/recent_topics_ui.js

```diff
@@ -807,6 +805,7 @@ export function complete_rerender() {
         is_spectator: page_params.is_spectator,
     });
     $("#recent_topics_table").html(rendered_body);
+    show_selected_filters();
```

Comment:
Can you explain the type bugs in a bit more detail? The commit message might be fine, but I could also imagine it making sense to have a comment just above this line noting that this needs to happen after recent topics is rendered because ___.